.github/workflows/globalworkflow-ci-pw-centos.yaml

```diff
@@ -15,28 +15,25 @@ on: [workflow_dispatch]
 #         └── ${pslot}
 env:
   TEST_DIR: ${{ github.workspace }}/${{ github.run_id }}
-  MACHINE_ID: orion
+  MACHINE_ID: noaacloud
 
 jobs:
   checkout-build-link:
```

Comment:
Can we have 3 steps? `checkout`, `build` and `link` OR 2 `checkout` and `build_link`.